inlang/source-code/cli/src/commands/module/build/command.ts

```diff
@@ -8,6 +8,22 @@ import { moduleBuildOptions } from "./moduleBuildOptions.js"
  */
 import { context } from "esbuild-wasm"
 
+// Define the forbidden Node.js imports
+const forbiddenNodeImports = ['fs', 'path', 'os', 'net']
+
+// Create an esbuild plugin to check Node.js imports
+const nodeAPICheckerPlugin = {
+	name: 'node-api-checker',
+	setup(build) {
+	  build.onResolve({ filter: /./ }, (args) => {
```

Comment:
And here too, see the CI pipeline:

<img width="944" alt="Screenshot 2023-11-16 at 12 30 34 PM" src="https://github.com/inlang/monorepo/assets/92092993/5a0e94af-60fb-43d7-aa0c-f20aa6458650">

Reply:
Link to CI: https://github.com/inlang/monorepo/actions/runs/6886826790/job/18739877557